src/services/shortLink/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Context, GraaspPlatform } from '@graasp/sdk';
+
+import { CLIENT_HOSTS, CLIENT_PREFIX } from '../../utils/config';
+
+export function getPlatformHost(shortLinkView: GraaspPlatform) {
+  const [BUILDER_HOST, PLAYER_HOST, LIBRARY_HOST] = CLIENT_HOSTS;
+
+  switch (shortLinkView) {
+    case GraaspPlatform.BUILDER:
+      return BUILDER_HOST;
+    case GraaspPlatform.LIBRARY:
+      return LIBRARY_HOST;
+    case GraaspPlatform.PLAYER:
+      return PLAYER_HOST;
+    default:
+      throw new Error('Invalid Shortlink view');
+  }
+}
+
+export function getUrlHost({ url }: { name: Context; url: URL }) {
+  const urlStr = url.toString();
+  // if the url ends with /, remove it
+  if (urlStr.endsWith('/')) {
+    return urlStr.slice(0, -1);
+  }
+  return urlStr;
+}
```

Comment:
Not sure what the purpose of this is, simply removing the trailing slash from origins ?